docs/community/tutorials-contest/2022/avax-subnet-customization/README.md

```diff
@@ -0,0 +1,975 @@
+
+#### `txAllowListConfig`
+
+This configures allows you to specify which addresses are authorized to make transactions. It's especially useful if you're building a private network.
+If you'd like to restrict the transaction sender to a specific list of addresses, you can set the `txAllowListConfig` to a JSON object with the following properties:
+
+```json
+"txAllowListConfig": {
+  "blockTimestamp": 0,
+  "adminAddresses": ["0x8db97C7cEcE249c2b98bDC0226Cc4C2A57BF52FC","0x0000...", "... and more"]
+}
+```
+
+Admin addresses can add new Admin and Allowed addresses. Precompiled contract is deployed to `0x0200000000000000000000000000000000000002`. [You can find more information from the Avalanche docs.](https://docs.avax.network/Subnets/customize-a-Subnet#restricting-who-can-submit-transactions)
+
+### Genesis Block
+This configuration is used to define the genesis block header. You generally don't need to change these. (except the `gasLimit` field) 
+
+#### `nonce`
+The result of the mining process iteration is this value. It can be any value in the genesis block, which is commonly set to `0x0`.
+
+#### `timestamp`
+The timestamp of the creation of the genesis block.
+
+#### `extraData`
+Optional extra data that can be included in the genesis block. This is commonly set to `0x`.
+
+#### `gasLimit`
+The total amount of gas that can be used in a single block. It should be set to the same value as in [the fee config](#fee-config). The value `0x7a1200` is hexadecimal and it's equal to `8,000,000`.
+
+#### `difficulty`
+The difficulty level applied during the nonce discovering of this block. It is usually set to `0x0` for the genesis block.
+
+#### `mixHash`
+[Quotation from Explanation of genesis file](#resources)  
+> The combination of nonce and `mixHash` must satisfy a mathematical condition described in the Yellowpaper, 4.3.4. Block Header Validity, (44). It allows to verify that the Block has really been cryptographically mined, thus, from this aspect, is valid. 
+
+This isn't so important for the genesis block. You can set it to `0x0000000000000000000000000000000000000000000000000000000000000000`.
+
+#### `coinbase`
+This is the address of the miner who mined the genesis block. It is usually set to `0x0000000000000000000000000000000000000000` for the genesis block.
+
+#### `parentHash`
+This is the Keccak 256-bit hash of the entire parent block’s header. It is usually set to `0x0000000000000000000000000000000000000000000000000000000000000000` for the genesis block.
+
+#### `gasUsed`
+This is the amount of gas used by the genesis block. It is usually set to `0x0`.
+
+#### `number`
+This is the number of the genesis block. It is usually set to `0x0`.
+
+#### `airdropHash` `baseFeePerGas` `airdropAmount`
+You can remove or leave these fields as they are. I'd recommend removing them. I'm not sure what they do.
+
+### Native Token Allocation
```

Comment:
Per [Markdownlint](https://github.com/DavidAnson/markdownlint) Headings should be surrounded by blank lines.

> MD022/blanks-around-headings/blanks-around-headers: Headings should be surrounded by blank lines [Expected: 1; Actual: 0; Below]

[More info: MD022](https://github.com/DavidAnson/markdownlint/blob/v0.25.1/doc/Rules.md#md022---headings-should-be-surrounded-by-blank-lines)